src/rhub/lab/model.py

```diff
@@ -289,6 +291,9 @@ def quota(self):
 
     def to_dict(self):
         data = super().to_dict()
+        data['region_name'] = self.region.name
+        data['user_name'] = get_user(self.user_id)[0]['username']
```

Comment:
You should use KeycloakClient directly

```
        data['user_name'] = get_keycloak().user_get(self.user_id)['username']
```

and similarly for groups.

API endpoint returns objects for use by the server (ConnexionResponse, Werkzeug response, ...), which doesn't make sense to use somewhere in the model.